expressions/variables.md

```diff
@@ -67,9 +75,64 @@ Fields have the same lifetime as the object they're in, rather than being scoped
 
 If the name of a field starts with `_`, it's __private__. That means only the type the field is in can have code that reads or writes that field. Otherwise, the field is __public__ and can be read or written from anywhere.
 
-Just like local variables, fields can be `var` or `let`. They can also have an initial value assigned in their definition, just like local variables, or they can be given their initial value in a constructor.
+Just like local variables, fields can be `var` or `let`. Nevertheless, rules for field assignation differ a bit from variable assignation. No matter the type of the field (either `var` or `let`), either:
```

Comment:
Use "assignment" instead of "assignation".

Same thing on lines 99, 101, 115.

Reply:
Done, fixed in my latest commit